internal/driver/config.go

```diff
@@ -124,7 +125,7 @@ func init() {
 	// take on one of a bounded set of values.
 	choices := map[string][]string{
 		"sort":        {"cum", "flat"},
-		"granularity": {"functions", "filefunctions", "files", "lines", "addresses"},
+		"granularity": {"functions", "filefunctions", "files", "lines", "columns", "addresses"},
```

Comment:
Would it make sense to instead make this an option like `columns` similar to `noinlines`? But I guess `pprof -lines -columns` would look weird.

Reply:
We should remove it from the granularity list here then?